### ISSUE TITLE
Not all subtitles are found.

### ISSUE THREAD

**Issue by jeffchen86** [pannal/Sub-Zero.bundle#553]
Hi,
I intend to download Chinese and English subtitles for all of our TV Shows and Movies. Chinese as the first option while English goes secondary, this language setting is preferred by my mum and wife.

However, there have been only some that were downloaded automatically. I keep tweaking configuration but am still unable to get it right... I only learn that I am able to 'search for missing subtitles (in recently-added items, max-age: 12 weeks)'. I tried it waited for few hours, it doesn't seem to make too much difference either. 

Attached is the log that was just pulled out of the 'Advanced functions' > 'Get my logs'. Can someone please take a look see what I'm missing here please?

[Logs_180822_20-49-12.zip](https://github.com/pannal/Sub-Zero.bundle/files/2310006/Logs_180822_20-49-12.zip)

**Comment by jeffchen86** [comment]
By the way, I often use Sub HD from Kodi. Not sure if there's plan to include it in SZ?

**Comment by jeffchen86** [comment]
[Logs_180823_10-56-54.zip](https://github.com/pannal/Sub-Zero.bundle/files/2312440/Logs_180823_10-56-54.zip)
Attaching logs after 14 hours when scheduled auto finding missing subtitles should have occurred. No change found in my library.

**Comment by pannal** [comment]
https://github.com/pannal/Sub-Zero.bundle/wiki/FAQ#q2-im-running-the-plugin-for-the-first-time-and-want-to-have-subtitles-for-all-of-my-old-items

**Comment by jeffchen86** [comment]
I tried it for few times through the past couple of weeks. I’ll try this one more time and provide update here.

**Comment by pannal** [comment]
Attach the log a couple of minutes after you triggered a metadata refresh.

**Comment by jeffchen86** [comment]
Oh does it have to be “a coupled of minutes” after I triggered it? It has been more than 1 hour and I checked no difference made. However, I checked the history of last auto updated (every 12 hours), few latest episodes downloaded last week now have subtitles, but still missing quite a bit in my library. I know I’ve maxed out the OpenSubtitles quota today so the ones downloaded were from assrt.net and addic7ted.

Let me attach the log after “a couple of minutes” when I’m about to trigger a new one in few hours.

**Comment by jeffchen86** [comment]
[Logs_180824_13-36-32.zip](https://github.com/pannal/Sub-Zero.bundle/files/2316988/Logs_180824_13-36-32.zip)
Here you go - logs generated 2-3 mins when I triggered the metadata refresh. It's still in progress not completed I think.

**Comment by jeffchen86** [comment]
I guess this is it?

**Comment by pannal** [comment]
Humm, you've got connection errors to sonarr/radarr:
`ConnectionError: HTTPConnectionPool(host='10.0.0.67', port=8989): Max retries exceeded with url: /api/episode?seriesId=16 (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x68d14710>: Failed to establish a new connection: [Errno 111] Connection refused',))
`
That may result in not all subtitles to be properly found.

**Comment by jeffchen86** [comment]
This is because Assrt.net has limit - 20 hits per minutes. How can I configure it to interval at 20 tries each minute?

**Comment by pannal** [comment]
No it isn't. This is your radarr/sonarr returning `[Errno 111] Connection refused`. This is not a provider error.

**Comment by jeffchen86** [comment]
Ok. I've removed all details of Sonarr and Radarr. Renaming method is set to Filebot. Let's see how different this time it'll be.

**Comment by pannal** [comment]
Well, there are no filebot attributes on `/EXTERNAL/TV SHOWS` as the logs indicate. How do you rename your media?

**Comment by jeffchen86** [comment]
/EXTERNAL is my Samba drive. I normally manually download via Sonarr (70%) and my PC (30%) depends which one comes easier for me. When it’s from PC, I used Filebot on PC.

**Comment by pannal** [comment]
OK, then you should enable all of those. Sonarr/Radarr/Filebot to make this work.

Also make sure that you don't move/rename the media managed by Sonarr/Radarr outside of those tools in any way.

Filebot doesn't work via shares.

**Comment by jeffchen86** [comment]
I've configured again to Sonarr/Radarr/Filebot, but this setting goes back to what it was 7 days ago.
FYI - My Sonarr/Radarr is linked to Jackett. This maybe the cause of API limit excesses.

**Comment by pannal** [comment]
No, again, Sonarr/Radarr refuses your requests: ```ConnectionError: HTTPConnectionPool(host='10.0.0.67', port=8989): Max retries exceeded with url: /api/episode?seriesId=16 (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x68d14710>: Failed to establish a new connection: [Errno 111] Connection refused',))
``` 

Please ensure that SZ is able to connect to `10.0.0.67:8989`.

**Comment by jeffchen86** [comment]
I'll upload a new log tonight. I found out a couple days back, Sonarr/Radarr/Jackett was crashing randomly on my pi 3 b+. This might be the reason. I'm trying different things to automate reviving these, but will get the logs when I'm sure at the time refreshing metadata, I have everything started correctly.

**Comment by jeffchen86** [comment]
[Logs_180829_11-57-40.zip](https://github.com/pannal/Sub-Zero.bundle/files/2330421/Logs_180829_11-57-40.zip)
New logs are attached.

**Comment by Cyriltra** [comment]
Sorry to bump this issue, but does assrt.net still works for SZ ?
When doing manual search for a tv show on assrt.net, I can find matches, but when do it via SZ, nothing is found
Thanks

**Comment by pannal** [comment]
@Cyriltra can you provide DEBUG logs after refreshing an item?

**Comment by Cyriltra** [comment]
@pannal Hi,here are the logs attached 
[com.plexapp.agents.subzero2.log](https://github.com/pannal/Sub-Zero.bundle/files/2775299/com.plexapp.agents.subzero2.log)
[Uploading com.plexapp.agents.subzero1.log…]()
[Uploading com.plexapp.agents.subzero.log…]()

**Comment by pannal** [comment]
@Cyriltra please set SZ's logging verbosity to DEBUG (at the bottom of SZs settings), then refresh an item, then post your logs again.

**Comment by Cyriltra** [comment]
@pannal ah yes I forgot to select DEBUG in the dropdown menu.
Here you go:
[com.plexapp.agents.subzero.log](https://github.com/pannal/Sub-Zero.bundle/files/2776342/com.plexapp.agents.subzero.log)

Thank you

**Comment by pannal** [comment]
The log looks somewhat OK. Is this still an issue?
